targets/ramips-mt7621

```diff
@@ -35,9 +35,7 @@ fi
 device ubnt-erx ubnt-erx
 packages '-hostapd-mini'
 factory
-sysupgrade '.tar'
```

Comment:
Why is the sysupgrade of ramips-mt7621 touched in that context?

Reply:
Because the patchset was rebased against next, not master.